src/main/webapp/resources/fonts/glyphicons-halflings-regular.svg

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
You don't need to add all font-awesome files to repo. Just add proper line to dependencies in bower.json file (e.g. "fontawesome": "4.3.0" - don't know which version exactly you have used). Then execute 'bower install' and add proper paths to index.html.